src/privileged/subscripts/PanelUI.jsm

```diff
@@ -0,0 +1,96 @@
+/* globals XUL_NS */
+
+function PanelUI(doc) {
+  this.site = null;
+  this.doc = doc;
+
+  const box = doc.createElementNS(XUL_NS, "vbox");
+
+  let elt;
+
+  elt = doc.createElementNS(XUL_NS, "description");
+  elt.appendChild(doc.createTextNode(this.getString("fxmonitor.popupHeader")));
+  elt.classList.add("headerText");
+  box.appendChild(elt);
+
+
+  elt = doc.createElementNS(XUL_NS, "description");
+  elt.classList.add("popupText");
```

Comment:
Why are these classes and not ids?

Reply:
I prefer classes here because they can be namespaced using `#notification-popup[popupid=fmonitor]`